src/streamertester.js

```diff
@@ -117,18 +117,30 @@ class StreamerTester {
       return addObjects(ac, cv)
     }, stats[0])
     combined.success_rate /= stats.length
+    // combined.profiles_num = stats[0].profiles_num
+
+    // for combined stats, recalc success rate 
+    combined.succcess_rate = combined.downloaded_segments / ((combined.profiles_num + stats.length) * combined.total_segments_to_send) * 100
     return combined
   }
 
   static FormatStatsForConsole(stats) {
+    // stats.ShouldHaveDownloadedSegments = (model.ProfilesNum + 1) * stats.SentSegments
+    // stats.SuccessRate = float64(stats.DownloadedSegments) / ((float64(model.ProfilesNum) + 1) * float64(stats.SentSegments)) * 100
```

Comment:
Just curious - why do we leave this commented out code here?

Reply:
it's for reference on the stats data structure and types.

P.s. I'm guessing 😆

Reply:
Ooo that's useful!

Reply:
@angyangie No hard reason, just reference for myself for data structure and for the algorithm used to calculate success rate in `golang` code